src/main/java/org/osiam/client/OsiamConnector.java

```diff
@@ -799,6 +811,8 @@ public QueryBuilder createQueryBuilder(Query original) {
         private String authServiceEndpoint;
         private String resourceServiceEndpoint;
         private String clientRedirectUri;
+        private Integer connectTimeout;

```

Comment:
Absolutly, I set them all to Integer. I need this type to be able to [check if it's set or not](https://github.com/osiam/connector4java/pull/167/files#diff-e61d51b09a914e6b31732b23b8dc2161R175).